demonstrations/tutorial_vqe_qng.py

```diff
@@ -0,0 +1,465 @@
+r"""
+Accelerating VQEs with quantum natural gradient
+===============================================
+
+*Authors: Maggie Li, Lana Bozanic, Sukin Sim (ssim@g.harvard.edu)*
+
+.. meta::
+    :property="og:description": Accelerating variational quantum eigensolvers 
+        using quantum natural gradients in PennyLane.
+    :property="og:image": https://pennylane.ai/qml/_images/qng_example.png
+    
+This tutorial showcases how one can apply quantum natural gradients (QNG) :ref:`[1, 2]` 
+to accelerate the optimization step of the Variational Quantum Eigensolver (VQE) algorithm :ref:`[3]`. 
+We will implement two small examples: estimating the ground state energy of (1) a single-qubit VQE 
+problem, which we can visualize using the Bloch sphere, and (2) the hydrogen molecule. 
+    
+Before going through this tutorial, we recommend that readers refer to the 
+`QNG tutorial <https://pennylane.ai/qml/demos/tutorial_quantum_natural_gradient.html>`__ and
```

Comment:
Using the `:doc:` syntax to make internal links make the demo slightly more resilient in case of future URL changes :)
```suggestion
:doc:`QNG tutorial </demos/tutorial_quantum_natural_gradient>` and
```